cobbler/actions/reposync.py

```diff
@@ -223,7 +218,7 @@ def librepo_getinfo(self, dirname):
         try:
             h.perform(r)
         except librepo.LibrepoException as e:
-            utils.die("librepo error: " + dirname + " - " + e.args[1])
+            raise CX("librepo error: " + dirname + " - " + e.args[1])
```

Comment:
```suggestion
            raise CX("librepo error: " + dirname + " - " + e.args[1]) from e
```

It is common practice to attach the old exception to the new one according to my information. If you have a reason to not do it I am fine with it but then let's add a comment why we didn't do it in this specific case please.